pkg/handler/module_streams.go

```diff
@@ -0,0 +1,58 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/content-services/content-sources-backend/pkg/api"
+	"github.com/content-services/content-sources-backend/pkg/dao"
+	ce "github.com/content-services/content-sources-backend/pkg/errors"
+	"github.com/content-services/content-sources-backend/pkg/rbac"
+	"github.com/labstack/echo/v4"
+)
+
+type ModuleStreamsHandler struct {
+	Dao dao.DaoRegistry
+}
+
+func RegisterModuleStreamsRoutes(engine *echo.Group, rDao *dao.DaoRegistry) {
+	rh := ModuleStreamsHandler{
+		Dao: *rDao,
+	}
+
+	addRepoRoute(engine, http.MethodPost, "/snapshots/module_streams/search", rh.searchSnapshotModuleStreams, rbac.RbacVerbRead)
+
+}
+
+// searchSnapshotModuleStreams godoc
+// @Summary      List modules and their streams for snapshots
+// @ID           searchSnapshotModuleStreams
+// @Description  List modules and their streams for snapshots
+// @Tags         module_streams
+// @Accept       json
+// @Produce      json
+// @Param        body  body   api.SearchSnapshotModuleStreamsRequest  true  "request body"
+// @Param  	     uuids	path	[]string	true	"Snapshot IDs."
+// @Param  	     package_names	path	[]string	true	"Package Names"
```

Comment:
```suggestion
// @Param        body  body   api.SearchSnapshotModuleStreamsRequest  true  "request body"
```

These other params aren't needed since they are in the body